controllers/rack.go

```diff
@@ -232,6 +232,20 @@ func (r *SingleClusterReconciler) createRack(rackState *RackState) (
 		return nil, reconcileError(err)
 	}
 
+	// Create pod service if node network is used in network policy
```

Comment:
Service has to be created before creating the pod. Initcontainer uses the service to fetch the mapped port for the pod. Are you sure this flow is working?

Reply:
Services are always created before pod in the scaleup function not in createSTS func. Even in the current master service creation logic in the createSTS func was of no use as rack.Size passed was 0, so no for loop execution.
Removed that code block completely.

Reply:
Rename the func name to reflect the changes